packages/SymbioticController/composer.json

```diff
@@ -4,7 +4,8 @@
         "php": "^7.1",
         "nette/di": "^2.4",
         "nette/application": "^2.4",
-        "odbav-to/presenter-route": "^0.1"
+        "odbav-to/presenter-route": "^0.1",
```

Comment:
I have been thinking alot about this, does this package really need `odbav-to/presenter-route` to its life and to be able to work? It seems to me that answer is _NO_. This one can work as well with different implementation of router (even nette's default router) but in `presenter-route` package situation is different - it can not work without this package, and there is _NO OTHER_ package, that it would work with, than this one, so i would set it there as dependency.

My recommendation is to move `odbav-to/presenter-route` to dev dependencies for tests and set is as recommended, but do not require it explicitly. What do you think?